### PR TITLE
152 friendly connection error messages

### LINKED ISSUE #152: Friendly connection error messages
As part of #149 [CRAN Policies](https://cran.r-project.org/web/packages/policies.html) indicate that:

> Packages which use Internet resources should fail gracefully with an informative message if the resource is not available or has changed (and not give a check warning nor error). 

All connections to SIVIGILA servers should address the above

### PR DESCRIPTION
**Pull request by jpavlich**
Fixes #152 
I used `tryCatch` since it is the recommended way to catch errors in `httr2` https://httr2.r-lib.org/reference/req_error.html
and it is used in functions that should abort when the connection fails

### CODE REVIEW

**Comment by jpavlich** [comment]
@GeraldineGomez I will make some more changes before sending it again to you

File: R/import_data.R [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+#' query_event_year_path <- config::get(file =
+#'            system.file("extdata",
+#'            "config.yml",
+#'            package = "sivirep"),
+#'            "query_diseases_by_year_path")
+#' make_request(query_event_year_path)
+#' }
+#' @noRd
+
+make_request <- function(url) {
```
Comment (GeraldineGomez) [inline R/import_data.R:17]: Hi Jaime, is it possible to change the function's name to Spanish? I also had to change the scraping function to Spanish for the package's format.

File: R/import_data.R [hunk 2]
```diff
@@ -41,22 +96,27 @@ import_geo_cods <- function(descargar = FALSE) {
 #' @return Una list con las enfermedades y los años disponibles
 #' para su descarga desde los microdatos de SIVIGILA
 #' @examples
+#' \dontrun{
```
Comment (GeraldineGomez) [inline R/import_data.R:99]: Based on Hugo's recommendation, we have to change the usage of `dontrun` to `donttest`

**Review by GeraldineGomez — CHANGES_REQUESTED** [review]
Hi @jpavlich,

Thank you so much for tackling this issue 😊. I think the messages are very clear and friendly for the user. I made some minor comments.

File: R/import_data.R (hunk 1 above)
Comment (jpavlich) [reply R/import_data.R]: Done

File: R/import_data.R (hunk 2 above)
Comment (jpavlich) [reply R/import_data.R]: Done

**Review by GeraldineGomez — APPROVED** [review]

**Pull request merged**